account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -159,12 +165,18 @@ export function createAlchemySmartAccountClient({
       }
       return customMiddleware ? customMiddleware(struct, args) : struct;
     },
-    feeEstimator: feeEstimator ?? alchemyFeeEstimator(transport),
+    // TODO(jh): not actually sure what should be the priority here b/w the first two.
```

Comment:
not totally sure on what order we should coalesce things in for `feeEstimator` and `gasEstimator`.